### DISCUSSION TITLE
My  Private API agot leaked in GIT HUB which is removed now. Is it possible to find in GITHUB where and it was created?who was the owner.

### DISCUSSION THREAD

**Discussion by arunmaya01** [community/community#47774] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

My  Private API agot leaked in GIT HUB which is removed now. Is it possible to find in GITHUB where and it was created?who was the owner.

**Comment by Valbuena99** [comment]
You can search the commit logs of the repository where your private API was leaked to see if there is any information about when it was created and who created it. You can also search the "Contributors" section of the repository to see who has contributed to the repository.
If you can't find this information in the repository itself, you can try contacting GitHub Support for help finding this information. You may need to provide more details about the repository and private API in question for specific help.

Greetings✌️
